webpack.config.js

```diff
@@ -0,0 +1,14 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  target: 'node',
+  mode: 'development',
+  optimization: {
+    minimize: false
+  },
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'index.js',
+  },
+};
```

Comment:
Newline should be added

Reply:
Merged. I will create a v1.4 tag once I review the other PR's as well.